forum/profiles/models.py

```diff
@@ -1,3 +1,70 @@
+from django.core.validators import MinValueValidator
 from django.db import models
+from users.models import User
```

Comment:
Isn't is better to use get_user_model() instead of importing User because we will be using customized user model?
Django docs:
get_user_model()
Instead of referring to User directly, you should reference the user model using django.contrib.auth.get_user_model(). This method will return the currently active user model – the custom user model if one is specified, or User otherwise.

Reply:
Done )